workspaces/acr/catalog-info.yaml

```diff
@@ -1,10 +1,23 @@
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-component
 apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
-  name: backstage-plugin-acr-workspace
-  title: 'Azure Container Registry plugin for Backstage'
+  name: backstage-community-acr
+  title: Azure Container Registry plugin
   description: A Backstage plugin that displays information about your container images available in the Azure Container Registry
+  annotations:
+    github.com/project-slug: backstage/community-plugins
+    #github.com/team-slug: backstage-community/acr-plugin-maintainers
```

Comment:
I am assuming that this is commented out since there isn't an acr-plugin-maintainers team. Could we just set this to `github.com/team-slug: backstage/community-plugins-maintainers` similar to what you did for the jfrog plugin?

Reply:
Thanks, good idea, I've updated that (and will push it now)